File: app/src/main/java/com/example/myanimedata/api/JpgResult.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class JpgResult {
    @SerializedName("image_url")
    private final String imageUrl;

    @SerializedName("small_image_url")
    private final String smallImageUrl;

    @SerializedName("large_image_url")
    private final String largeImageUrl;

    public String getImageUrl() {
        return imageUrl;
    }

    public String getSmallImageUrl() {
        return smallImageUrl;
    }

    public String getLargeImageUrl() {
        return largeImageUrl;
    }

    public JpgResult(String imageUrl, String smallImageUrl, String largeImageUrl) {
        this.imageUrl = imageUrl;
        this.smallImageUrl = smallImageUrl;
        this.largeImageUrl = largeImageUrl;
    }
}

File: app/src/main/java/com/example/myanimedata/api/JpgResult.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class JpgResult(
    @field:SerializedName("image_url") val imageUrl: String? = null,
    @field:SerializedName("large_image_url") val largeImageUrl: String? = null
) : Parcelable